riotwatcher/_apis/LeagueApiV3.py

```diff
@@ -61,6 +61,21 @@ def by_summoner(self, region, summoner_id):
             '/lol/league/v3/leagues/by-summoner/{summonerId}'.format(summonerId=summoner_id)
         )
 
+    def leagues(self, region, league_id):
```

Comment:
I think the name would be better as "by_id", as this is more consistent with the naming in the other endpoints